packages/mobile/__mocks__/@celo/contractkit/index.ts

```diff
@@ -0,0 +1,45 @@
+import BigNumber from 'bignumber.js'
+
+export const newKitFromWeb3 = () => ({
```

Comment:
Would be cool if this lived in contract kit itself so other packages could use it. This file could just import and re-export the mock